README.md

```diff
@@ -66,18 +86,18 @@ EXAMPLES
   Executing script...
 ```
 
-_See code: [dist/commands/exec/index.ts](./src/commands/exec/index.ts)_
+_See code: [dist/commands/exec/index.ts](https://github.com/eclipse-velocitas/velocitas-cli/blob/v0.2.1/dist/commands/exec/index.ts)_
```

Comment:
Shouldn't we be cautious by using the direct version of the cli repo? That means we always need to manually change the versions when updating the cli or can we just use the oclif CLI to update the readme?

Reply:
This was auto generated by `oclif readme` it slipped through. I have now fixed `oclif readme`'s generated paths.